src/components/InspirationCard.jsx

```diff
@@ -0,0 +1,47 @@
+import Lottie from "lottie-react";
+import { useEffect, useState } from "react";
+import animation from "../assets/animations/Animation-planning.json";
+import "../styling/InspirationCard.css";
+
+const InspirationCard = () => {
+  //state variables
+  const [quote, setQuote] = useState([]);
+  const [loading, setLoading] = useState(true);
+  //Api url
+  const URL = "https://type.fit/api/quotes";
+
+  const fetchQuote = () => {
```

Comment:
We love how creative you are and how you implemented learnings from other projects in this weeks project!